docs/components/table.md

```diff
@@ -351,6 +351,57 @@ export default {
 </script>
 ```
 
+An alternative usage is with the native `key` attribute in conjuction with a `fetcher` and SWRV `revalidate` function. In this case you will want the `key` to start at `0`.
+Since the `fetcher` function will handle the intial GET of the data, we want the cache key for the `revalidate` function to evaluate to `falsey` on page load (to avoid an unnecessary duplicate call), and will manually call `revalidate()` and increment the `key` to trigger a refetch and redraw of the table.
```

Comment:
nit: Add an empty line between the paragraphs
```suggestion

Since the `fetcher` function will handle the intial GET of the data, we want the cache key for the `revalidate` function to evaluate to `falsey` on page load (to avoid an unnecessary duplicate call), and will manually call `revalidate()` and increment the `key` to trigger a refetch and redraw of the table.
```